src/Component/Footer.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable */
+import React from 'react';
+/* eslint-enable */
```

Comment:
- [ ] Try to resolve errors instead of disabling it because that is what `eslint` is here for to help you write clean codes.